### DISCUSSION TITLE
About tasks and calendar view (or integration)

### DISCUSSION THREAD

**Discussion by mcastillof** [TechbeeAT/jtxBoard#173] [Q&A]
I have tested various apps, and jtx Boards have all the features that I would like, but a calendar view or something like that.

Certain apps like Mozilla Thunderbird and Nextcloud show tasks that have a starting date or a due date in the calendar. I think this calendars are very helpful to show a the big picture of what events/tasks are starting or due. I would love that jtx Board had a calendar view or something like that.

Do you use another app that has a calendar to complement jtx Board? or do you use just filters/groups/sorts to have a picture of what is next in your agenda?

**Comment by patrickunterwegs** [comment]
Hey @mcastillof , personally I using filters, but I was thinking of a dedicated calendar option for some time. 
But to get it right, here you mean a calendar view in jtx Board where you can see e.g. journals appear on their specific date in the calendar or tasks to appear on their start (or due?) date in the calendar?
It is not about seeing your journals and tasks in another calendar app, right?

**Reply by mcastillof** [reply]
Hello @patrickunterwegs 

> but I was thinking of a dedicated calendar option for some time.

Do you mean a dedicated calendar option as in a separate application or a new activity in jtx Board?

> But to get it right, here you mean a calendar view in jtx Board where you can see e.g. journals appear on their specific date in the calendar or tasks to appear on their start (or due?) date in the calendar?

I was just thinking in a new calendar activity inside jtx Board where you can see your tasks that have date (start, due or both) **that have not been completed**, as in my workflow I'm only interested in uncompleted tasks. Now that you mention journals, maybe is a good idea to have a new calendar activity inside jtx Board, with 2 toggle buttons, for journals and tasks. By toggling them, it would display journals, tasks, or both. if someone is interested in tasks completed and uncompleted, then a third toggle button would be necessary.

> It is not about seeing your journals and tasks in another calendar app, right?
I would prefer a single app.

**Reply by patrickunterwegs** [reply]
I didn't mean another app, actually I just wanted to get a better understanding of your thoughts. I will think about possible solutions, but it will take a bit, there are still many things to do until I will tackle another big thing.
